### PR TITLE
Add predr.ag

### PR DESCRIPTION
**Pull request by yuvadm**
Hey @obi1kenobi, would love to add you to our exclusive club, just need your `title` :)

### CODE REVIEW

File: names/predr.ag.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+domain: predr.ag
+name: Predrag Gruevski
```
Comment (obi1kenobi) [inline names/predr.ag.yml:2]: How about this for a title?
```suggestion
name: Predrag Gruevski
title: Purveyor of fine dev tools
```

**Review by obi1kenobi — APPROVED** [review]
Thanks for making this PR and letting me know about the Domain Name Hack Club 🙌

**Comment by obi1kenobi** [comment]
Also, fun fact, I own `gruev.ski` too so `predrag.gruev.ski` is also me — I just haven't set up the redirect (yet!).

**Comment by yuvadm** [comment]
@obi1kenobi oh wow! I think you're the first person to own *two* domain hacks for his name :joy:

**Pull request merged**